Example/ResizableView.swift

```diff
@@ -0,0 +1,129 @@
+import UIKit
+
+class ResizableView: UIControl {
```

Comment:
Not sure there is any sense in keeping `ResizableView.swift`, `SettingsItems.swift`, `SettingsViewController.swift` at this point.